recipe/meta.yaml

```diff
@@ -39,11 +39,12 @@ requirements:
     - setuptools
     - cython >=0.29,<0.30
     - spdlog >=1.8.5,<2
+    - cuda-python >=11.5,<12.0
   run:
     - python
     - numba >=0.49
     - numpy
-    - {{ pin_compatible('cudatoolkit', max_pin='x', lower_bound='11.2') }} # cudatoolkit >=11.2,<12.0.0
+    - {{ pin_compatible('cuda-python', min_pin='x.x', max_pin='x.x') }}
```

Comment:
Typically `min_pin` [defaults to `'x.x.x.x.x.x'`]( https://github.com/conda/conda-build/blob/0a6efdcfa02484954e7a3fdf5c2ec4bb3cb6682e/conda_build/jinja_context.py#L203 ). If we want looser pinning than that, agree this makes sense

Reply:
Yeah I think either `x.x` or `x.x.x` should be sufficient.